src/components/MovieListPage/MovieListPage.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from 'react'
+import { GenreSelect } from '../GenreSelect/GenreSelect.tsx'
+import SortControl from '../SortControl/SortControl.tsx'
+import { MovieTile } from '../MovieTile/MovieTile.tsx'
+import { Search } from '../Search/Search.tsx'
+import { MovieGenres } from '../../common/constants/constants.ts'
+import './MovieListPage.css'
+import { MovieDetails } from '../MovieDetails/MovieDetails.tsx'
+import { getMovies } from '../../api/Movie.ts'
+import { Movie } from '../../common/interfaces/Movie'
+import { capitalizeFirstLetter } from '../../utils/utils.ts'
+
+const MovieListPage: React.FC = () => {
+    const [searchQuery, setSearchQuery] = useState('')
+    const [sortCriterion, setSortCriterion] = useState('')
+    const [activeGenre, setActiveGenre] = useState('ALL')
+    const [selectedMovie, setSelectedMovie] = useState<Movie | null>()
+    const [movieList, setMovieList] = useState<Movie[]>([])
+    const genres = ['ALL', ...MovieGenres].map(genre => genre.toUpperCase())
+
+    useEffect(() => {
+        getMovies({
+            search: searchQuery.trim(),
+            sortBy: sortCriterion,
+            searchBy: 'title',
+            sortOrder: 'asc',
```

Comment:
This is your app and you can do whatever you like, but maybe sorting movies from newest to oldest would be a better option (when the `sortCriterion` is `release_date` that is).

Reply:
Now when the sorting is by title, movies go from Z to A. I would say the order should be ascending for title sorting, and descending for release date sorting.